### PR TITLE
Add Commander option on Binary setup page, Closes #240

### LINKED ISSUE #240: Add Commander option to Core Binary setup
Lisk Commander v2.2.0 offers commands to install and manage Lisk Core Binary distribution with Lisk Commander.

Lisk Core Binary Setup should mention and explain the new option and point to the corresponding docs of Lisk Commander.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #240

### CODE REVIEW

File: setup/binary.md [hunk 1]
```diff
@@ -1,19 +1,80 @@
...
+
+To complete the installation some prerequisites need to be fulfilled.  If you have already performed these, please proceed to the [Installation](#installation) chapter. Please follow the instructions below to load the required software to your system.
+
+### Open the necessary ports
+
+To connect to the desired network with Lisk Core, please ensure that the corresponding ports are open:
+
+| Network | httpPort(HTTP) | wsPort(TCP) |
+| --------|----------------|-------------|
+| Mainnet | 8000           | 8001        |
```
Comment (ManuGowda) [inline setup/binary.md]: For the commander, the port can be different, based on the installation, I am not sure how to document that, but after the installation `lisk core:status NAME` will the details about port related to the specific installation.
@fchavant what's your thought on this? how can we convey it?

Comment (ghost) [reply setup/binary.md]: Mark this as relevant only installations performed with `installLisk.sh`?

File: setup/binary.md [hunk 2]
```diff
@@ -1,27 +1,73 @@
...
 
 ## Pre-Install
 
-This document will detail how to prepare a system for the installation of Lisk Core.  It will guide you through the installation of important dependencies, as well as user creation.
+To complete the installation some prerequisites need to be fulfilled.  If you have already performed these, please proceed to the [Installation](#installation) chapter.
 
 ### Determine if your platform can run Lisk Core
 
 ###### Supported Platforms
 - Ubuntu 18.04 x86_64
```
Comment (ghost) [inline setup/binary.md]:
```suggestion
- Ubuntu 18.04 (LTS) x86_64
```

File: setup/binary.md (hunk 1 above)
Comment (ghost) [reply setup/binary.md]: To be clear: users that installed Core using `lisk core:install` will still need to open up 2 ports but those are not static. They will need to find out which ports to open with `lisk core:status`.

**Review by ghost — CHANGES_REQUESTED** [review]

File: setup/binary.md (hunk 1 above)
Comment (Tschakki) [reply setup/binary.md]: So they can only know which ports to open, after they installed it?
Should I add this as post-installation step then?

Comment (ghost) [reply setup/binary.md]: > So they can only know which ports to open, after they installed it?

That is my understanding. @ManuGowda can confirm.

> Should I add this as post-installation step then?

Yes

Comment (ManuGowda) [reply setup/binary.md]: yes, the user can know the ports after installation using `status` command.

File: upgrade/binary.md [hunk 3]
```diff
@@ -3,7 +3,22 @@
...
+## Upgrade with Lisk Commander
+
+> __Note:__ The `core:upgrade` command is only supported for Lisk Core versions > `2.0.0` and will only work, if Lisk Core has been installed with Lisk Commander before.
+
+First, go into the directory, where your Lisk Core installation is located.
+
+Then, run the following command to upgrade Lisk Core:
+
+```bash
+lisk core:upgrade <location>
```
Comment (ghost) [inline upgrade/binary.md:15]: `lisk core:upgrade --help` prints:
```
USAGE
  $ lisk core:upgrade NAME
[...]
```

Comment (Tschakki) [reply upgrade/binary.md:15]: yes but `location` is more precise imo, as it is the name of the root folder of the node.

also, we explain precisely what is meant here in the line below.

**Review by ghost — APPROVED** [review]

**Pull request merged**